### DISCUSSION TITLE
Ideas for a public API

### DISCUSSION THREAD

**Discussion by baseTwo** [FirelyTeam/firely-cql-sdk#596] [Ideas]
We are planning the design of the public API around all the internals in the coming days (see #580 ). If you have any particular thoughts you'd like to share, feel free to do so here.

**Comment by and-hus** [comment]
@baseTwo 
I am interested to join the discussion around the public API topics listed in #580 , providing the perspective of a consumer of these APIs

**Reply by baseTwo** [reply]
Hi Andreas, 

Happy to hear your inputs on this page.

PS. since I'm going to be away from today until middle November, we did not go into this into much depth yet, but rather we will pick this up when I return.

**Reply by and-hus** [reply]
Perfect - let's pick up the discussion once you're back.
